indra/newview/scripts/lua/leap.lua

```diff
@@ -40,25 +40,25 @@
 
 local fiber = require('fiber')
 local ErrorQueue = require('ErrorQueue')
--- local debug = require('printf')
-local function debug(...) end 
+local function dbg(...) end 
```

Comment:
I know to little about lua  but ... I saw this snipped a few times now, would it not make sense to either define such a function in C++ or some global lua file, to have it everywhere and everywhere the same?

Reply:
I've spent enough time debugging these modules to want a quick way to turn debugging output on or off on a per-module basis. This incantation:
```
local function dbg(...) end
-- local dbg = require 'printf'
```
lets me turn on module-specific debugging output by merely deleting the `--` prefix.

Reply:
I'd still argue for a global function, taking as first argument the location it is coming from. (Much like the underused LL_XXX("where") )

Not only will this enrich the final logs, but also logging can then be tweaked on a "where are you from" basis